Firmware/mmu2.cpp

```diff
@@ -511,7 +511,7 @@ void MMU2::manage_response(const bool move_axes, const bool turn_off_nozzle) {
         // - still running -> wait normally in idle()
         // - failed -> then do the safety moves on the printer like before
         // - finished ok -> proceed with reading other commands
-        
+        manage_heater();
         // @@TODO this needs verification - we need something which matches Marlin2's idle()
```

Comment:
this @@TODO can be removed if `manage_heater()` and `manage_inactivity()` do their job